scan/lib/scan/test_command_generator.rb

```diff
@@ -54,7 +54,7 @@ def options # rubocop:disable Metrics/PerceivedComplexity
         options << "-scmProvider system"
       end
       if config[:result_bundle_path]
-        options << "-resultBundlePath '#{config[:result_bundle_path].shellescape}'"
+        options << "-resultBundlePath '#{config[:result_bundle_path]}'"
```

Comment:
I think this fix could lead to issues if the `result_bundle_path` contains single-quotes in its path, right?

So I think the better solution would probably be to keep the `shellescape`, but to remove the single quotes 🤔 

```suggestion
        options << "-resultBundlePath #{config[:result_bundle_path].shellescape}"
```

Reply:
@AliSoftware Thanks a lot for the suggestion. I also agree that it’s a better solution. I have updated the PR.